src/app/api/checkout/route.ts

```diff
@@ -0,0 +1,79 @@
+// NOTE: NextRequest and NextReponse will only work for middleware, and not for 
+// pages or components, so there is a specific command line for it.
+// eslint-disable-next-line
+import { NextRequest, NextResponse } from "next/server";
+import Stripe from 'stripe';
+
+// NOTE: Instead using `require()`, should use `Stripe()`
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY!)
+
+
+// eslint-disable-next-line
+function GET(req: NextRequest, res: NextResponse) {
+    return new Response(JSON.stringify({ code: "Hello Yooshie" }), {
+        status: 200
+    })
+}
+
+async function handler(req: NextRequest, res: NextResponse) {
```

Comment:
Decide if you want to use dedicated GET, POST etc. functions, or if you want to use a generic handler. Don't mix both methods